data/boarding missions.txt

```diff
@@ -23,6 +23,118 @@ mission "Assisting Merchant"
 	# A mission with no destination will not be offered, so give it one.
 	# What the destination is doesn't matter, because you always 'decline' this.
 	destination Earth
+	
+mission "Assisting Pirates (Thief)"
+	assisting
+	repeat
+	to offer
+		random < 12
+	source
+		government Pirate
+	on offer
+		payment -100000
+		conversation
+			`After assisting the <origin> you find that 100000 credits have been removed from your account.`
+				decline
+	destination Greenrock
+
+mission "Assisting Pirates (Setback)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government Pirate
+	on offer
+		payment 100
+		conversation
+			`The captain of the <origin> gives you <payment> for "helping him out of a temporary setback."`
+				decline
+	destination Greenrock
+	
+mission "Assisting Syndicate"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment>.`
+				decline
+	destination Foundry
+	
+mission "Assisting Oathkeepers"
```

Comment:
Why not merge Oathkeeper and Navy assisting missions?

Reply:
Due to them being relatively independent of each other and that one body has a higher view of you than the other.

Reply:
>Due to them being relatively independent of each other

The Oathkeepers are a separate branch under the Navy, not an independent body.

>and that one body has a higher view of you than the other.

Forward compatibility issue: this assumes that the player is liked by the Oathkeepers in all branches. Also, I wouldn't even say that the Oathkeepers have a higher view of the player in the Checkmate route.